model/registry/container.yaml

```diff
@@ -95,3 +95,18 @@ groups:
         brief: >
           Container labels, `<key>` being the label name, the value being the label value.
         examples: [ 'container.label.app=nginx' ]
+      - id: cpu.state
+        brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
```

Comment:
I may have commented on this above but not sure: Here is the attribute registry, which is meant to allow sharing attributes across signals. But the `brief` here specifically talks about `data point` which is a metric concept. Such term does not exist in logs or spans. 

I'd think it's better if this is generic and whenever this is used, one can overwrite the brief with the specific use case.

Reply:
This was also discussed at https://github.com/open-telemetry/semantic-conventions/pull/681/files#r1476038475, right?

I can change it for now, but sooner or later we need to consider unifying the `*.cpu.state` attribute I think. I will file an issue to discuss this.